### ISSUE TITLE
renaming note just adds note with new name

### ISSUE THREAD

**Issue by ingvald** [oleeskild/digitalgarden#173]
git repo didn't remove old name versions, causing build error for the dg-home note
(output conflict, multiple input files writing to index file)

perhaps either some fix or some docs guidance is desirable?

**Comment by ancillarymagnet** [comment]
Yeah chiming in with steps to reproduce in case it's unclear since I ran into this myself - agree that better guidance in docs is minimum fix:

1. Publish a note with dg-home = true. 
2. Change the name of the note
3. Publish the newly-renamed note with Publish Single Note command
4. Vercel build fails with error 1 and an opaque message

The issue is that you now have two published notes with dg-home=true. The solve is to Publish Multiple Notes, which will 'publish the deletion' of the note with its old name, thus removing it.

Thanks for the project, hope this is helpful.
